src/app/system/manage-surveys/create-survey/create-survey.component.html

```diff
@@ -0,0 +1,163 @@
+              <input matInput maxlength="255" required formControlName="text">
+              <mat-error *ngIf="question.controls.text.hasError('required')">
+                Text is <strong>required</strong>
+              </mat-error>
+            </mat-form-field>
+
+            <mat-form-field fxFlex="98%">
+              <mat-label>Description</mat-label>
+              <textarea matInput formControlName="description"></textarea>
+            </mat-form-field>
+
+            <mat-divider id="margined-divider" [inset]="true"></mat-divider>
+
+            <div fxFlexFill fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column">
+
+              <div fxFlex="98%" fxLayout="row" fxLayoutAlign="space-between center">
+                <h4 class="mat-h4">Options</h4>
+                <button type="button" mat-raised-button color="primary" (click)="addResponse(questionIndex)">
+                  <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+                  Add Option
+                </button>
+              </div>
+
+              <div fxFlexFill formArrayName="responseDatas" *ngFor="let responseData of getResponseDatas(questionIndex).controls; let responseIndex = index;">
+                
+                <div fxFlexFill fxLayout="row wrap" fxLayoutGap="2%" [formGroupName]="responseIndex">
+
+                  <mat-form-field fxFlex="43%">
+                    <mat-label>Text</mat-label>
+                    <input matInput required formControlName="text">
+                    <mat-error *ngIf="responseData.controls.text.hasError('required')">
+                      Text is <strong>required</strong>
+                    </mat-error>
+                  </mat-form-field>
+
+                  <mat-form-field fxFlex="43%">
+                    <mat-label>Value</mat-label>
+                    <input matInput required formControlName="value">
+                    <mat-error *ngIf="responseData.controls.value.hasError('required')">
+                      Value is <strong>required</strong>
+                    </mat-error>
+                    <mat-error *ngIf="responseData.controls.value.hasError('pattern')">
+                      Value <strong>must be an integer between -9999 and 9999</strong>
+                    </mat-error>
+                  </mat-form-field>
+
+                  <div fxFlex="8%">
+                    <div class="delete-wrapper">
+                      <button type="button" color="warn" mat-icon-button matTooltip="Delete" matTooltipPosition="above" (click)="removeResponse(getResponseDatas(questionIndex), responseIndex)" [disabled]="getResponseDatas(questionIndex).controls.length === 1">
+                        <fa-icon icon="trash" size="lg"></fa-icon>
+                      </button>  
+                    </div>
+                  </div>
+                        
+                </div>
+
+              </div>
+
+            </div>
+
+          </div>
+
+        </mat-card-content>
+
+        <div>
```

Comment:
Required, or else there will be a style applied to the mat-card-actions which removes the margins, making the form look quite poor.